library/std/src/sys/windows/net.rs

```diff
@@ -26,26 +26,31 @@ pub mod netc {
 
 pub struct Socket(c::SOCKET);
 
-static INIT: Once = Once::new();
+static WSA: SyncOnceCell<unsafe extern "system" fn() -> i32> = SyncOnceCell::new();
```

Comment:
Could this be named `WSA_CLEANUP`? Or otherwise wrap the `fn` pointer in a `struct` - as it stands, the `WSA` name doesn't make it obvious what the `fn` pointer is supposed to be.